go.mod

```diff
@@ -31,3 +31,5 @@ replace github.com/golangci/lint-1 v0.0.0-20180610141402-ee948d087217 => github.
 replace github.com/timakin/bodyclose => github.com/golangci/bodyclose v0.0.0-20190714144026-65da19158fa2
 
 replace mvdan.cc/unparam v0.0.0-20190124213536-fbb59629db34 => mvdan.cc/unparam v0.0.0-20190209190245-fbb59629db34
+
+replace golang.org/x/sys => golang.org/x/sys v0.0.0-20220811171246-fbc7d0a398ab
```

Comment:
Why this change?

Reply:
> Why this change?

@micahhausler This to fix `undefined: unsafe.Slice` errors introduced by the `aws-sdk-go` version bump up. The `unsafe.Slice` was added as part of Go 1.17, we are using Go 1.15 in `aws-encryption-provider`. I took the accept answer in https://stackoverflow.com/questions/74049351/gcloud-functions-deploy-go-runtime-error-undefined-unsafe-slice-error-id-2f5

Reply:
Go 1.15 is unmaintained... why don't we bump the Go version to something newer. I think you can just bump it in this PR